packages/react/src/Button/Button.module.css

```diff
@@ -113,3 +113,17 @@
 .Button--disabled:not(.Button--secondary) {
   box-shadow: none;
 }
+
+@media (forced-colors: active) {
```

Comment:
TIL. Thanks for adding this, I suspect we'll probably need to do the same in a few other components 😓

Reply:
No worries! I think it'll be a case by case basis for now. I'll have to do some research to see if there's a more global way we can apply these rules.